ci_framework/roles/build_openstack_packages/tasks/install_dlrn.yml

```diff
@@ -0,0 +1,158 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Install epel-release repo
+  become: true
+  ansible.builtin.package:
+    state: present
+    name: epel-release
```

Comment:
shouldn't we disable EPEL and activate it only for specific subset of package installations?